### PR TITLE
[18.06] https-dns-proxy: update binary to 2020-08-21

### PR DESCRIPTION
**Pull request by stangri**
Maintainer: me
Compile tested: mvebu, WRT3200ACM, 18.06.8/19.07.4
Run tested: mvebu, WRT3200ACM, 18.06.8, start/stop, query DoH servers

Description: this updates to the latest release binary and contains cosmetic fixes to README and init script.

PS. I'm also requesting merge rights to packages.

Signed-off-by: Stan Grishin <stangri@melmac.net>

### CODE REVIEW

**Comment by BKPepe** [comment]
Is this a security fix or why this is submitted against 18.06 branch?

**Comment by stangri** [comment]
There's a pesky issue of 100% CPU spikes on some systems and there's a hope this new binary will fix it.

File: net/https-dns-proxy/files/README.md [hunk 1]
```diff
@@ -1,94 +1,3 @@
-# DNS Over HTTPS Proxy (https-dns-proxy)
+# README
```
Comment (BKPepe) [inline net/https-dns-proxy/files/README.md:1]: This is no go for me. Sorry! This is almost breaking change in OpenWrt 18.06.

File: net/https-dns-proxy/test.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/etc/init.d/"$1" version 2>&1 | grep "$2"
```
Comment (BKPepe) [inline net/https-dns-proxy/test.sh:3]: Why? Test.sh is not used in OpenWrt 18.06 branch.

**Review by BKPepe — CHANGES_REQUESTED** [review]

**Review by stangri — COMMENTED** [review]
To maintain same code across branches. This has already been accepted for the other packages I maintain.

**Pull request merged**